iothub/device/src/Transport/Amqp/AmqpConnectionHolder.cs

```diff
@@ -137,6 +137,7 @@ private async Task<AmqpConnection> EnsureConnection(TimeSpan timeout)
                     AmqpConnection.Closed += OnConnectionClosed;
                     if (Logging.IsEnabled) Logging.Associate(this, AmqpConnection, $"{nameof(AmqpConnection)}");
                     if (Logging.IsEnabled) Logging.Associate(this, AmqpCbsLink, $"{nameof(AmqpCbsLink)}");
+                    EventCounterLogger.GetInstance().OnAmqpConnectionEstablished();
```

Comment:
Let's follow the Logging pattern for clarity:
`if (EventCounters.IsEnabled) EventCounters.<...>`